### PR TITLE
Compatibility - Java

### PR DESCRIPTION
**Pull request by daschl**
Note that this is still pending unification/consistency by PM.
ping @ingenthr @ToddGreenstein @simonbasle

### CODE REVIEW

File: content/sdk/java/compatibility.dita [hunk 1]
```diff
@@ -75,90 +80,123 @@
...
 
         </body>
     </topic>
+
+  <topic id="jdk-compat">
+    <title>JDK Version Compatibility</title>
+    <shortdesc>The Java SDK is tested with the Oracle JDK which should be used for best experience and supportability. Other JDK implementations might
+      work but are  not tested and are unsupported.</shortdesc>
+    <body>
+    <p>The folloing Oracle JDKs are supported for <b>Java SDK 1.4 and 2.0 - 2.3</b>:</p>
```
Comment (simonbasle) [inline content/sdk/java/compatibility.dita]: typo "following"

File: content/sdk/java/compatibility.dita [hunk 2]
```diff
@@ -57,13 +57,18 @@
                         </row>
                         <row>
                             <entry><b>Server 4.0</b></entry>
-                            <entry namest="c2" nameend="c5">Compatible</entry>
-                            <entry><b>Recommended</b></entry>
+                            <entry namest="c2" nameend="c4">Compatible</entry>
```
Comment (simonbasle) [inline content/sdk/java/compatibility.dita]: I wonder if the server version lines makes sense now, they all display the same information (Compatible up to SDK 2.2, Preferred for SDK 2.2 and Recommended for 2.3). Maybe we can just use this table to say which SDK are archived / have bugfix releases / have active development? I think I remember @ToddGreenstein wasn't convinced by the compat/preferred/recommended terminology...

**Comment by simonbasle** [comment]
👍 looks good to me knowing that @ToddGreenstein will harmonize all compatibility matrices in the future ;)

**Comment by ingenthr** [comment]
Rebased and merged.

**Pull request closed without merging**